README.md

```diff
@@ -287,6 +288,23 @@ const DoublerNums = flow(
 )
 ```
 
+### `addDefaultProps()`
+
+```js
+addDefaultProps(
+  defaults: (incomingProps: Object) => Object | Object,
+): Function
+```
+
+Adds some props only if they are not already present in the incoming props (ie. if they are `null` or `undefined`).
+
+For example:
+
+```js
+addDefaultProps({ a: 1, b: 2 })({ a: "one" })
```

Comment:
Let's make the example more "realistic" like the other examples

So (a) showing usage of the helper in the context of an example component (b) using somewhat more "real" prop names than `a`/`b`

In this case since `addDefaultProps()`'s behavior is dependent on how the component is called, the example should show examples of the component (that uses `addDefaultProps()` being called in ways that kicks in the default props and that doesn't. See eg the `addProps()` docs for a somewhat similar example (with `Doubler` + `Outer`)

Eg
```
const Greeting = flow(
  addDefaultProps({
    name: 'world',
  }),
  ({name}) => <div>Hello, {name}!</div>
)

const AnonymousGreeting = () =>
  <Greeting /> // renders "Hello, world!"

const ReactGreeting = () =>
  <Greeting name="React" /> // renders "Hello, React!"
```